mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(product) {
```

Comment:
Did this work? It looks like you are adding the tax to the original amount, but this should only be working out the tax. You should be adding the tax to the original amount later in the code.

Reply:
Apologies Melese - you were right in the first place! You are supposed to calculate the full amount here - rather than just tax. I misunderstood the requirement.